src/App.js

```diff
@@ -61,14 +69,11 @@ class ColorApp extends Component {
   // }
 
   render()  {
-    const colors = this.state.color.map((color, i)=>{
+    const colors = this.state.colors.map((color, i)=>{
       return <Colors 
-        hex={color.color} 
-        color={color.color}
-        lock={color.lock}
+        colorObj = {color}
         key={i}
         toggle={()=>this.lockButtonToggle(i)}
-        
         />
```

Comment:
Line 66-68 

You were adding in more props that you need to. This can clutter you component with too many pieces to keep track of. The color object in the map parameter already contains what you need. No need to separate them.